app/Helpers/Helper.php

```diff
@@ -40,3 +42,12 @@ function durationToSpan($duration): string {
 
     return $return;
 }
+
+function userTime(null|Carbon|\Carbon\Carbon|string $time=null, ?string $format=null, bool $iso=true): string {
```

Comment:
see review comment above in line 3

```suggestion
function userTime(null|\Carbon\Carbon|string $time=null, ?string $format=null, bool $iso=true): string {
```